File: app/src/main/java/com/june/mvpkj/j/main/MainActivity.java
package com.june.mvpkj.j.main;

import android.view.View;
import com.june.mvpkj.databinding.ActivityMainJBinding;
import com.june.mvpkj.j.base.BaseActivity;

import java.util.ArrayList;
import java.util.List;

public class MainActivity extends BaseActivity<MainPresenter> implements MainContract.View {

    private ActivityMainJBinding binding;

    @Override
    protected MainPresenter createPresenter() {
        return new MainPresenter(this);
    }

    @Override
    protected View layout() {
        binding = ActivityMainJBinding.inflate(getLayoutInflater());
        return binding.getRoot();
    }

    @Override
    protected void init() {

    }

    @Override
    public void successResponse() {

    }

    @Override
    public void errorResponse() {

    }
}

File: app/src/main/java/com/june/mvpkj/k/main/MainActivity.kt
package com.june.mvpkj.k.main

import android.view.View
import com.june.mvpkj.databinding.ActivityMainBinding
import com.june.mvpkj.k.base.BaseActivity

class MainActivity : BaseActivity<MainContract.View,MainPresenter>(),MainContract.View {

    private val binding: ActivityMainBinding by lazy { ActivityMainBinding.inflate(layoutInflater) }

    override fun createPresenter(): MainPresenter {
        return MainPresenter(this)
    }

    override fun layout(): View {
        return binding.root
    }

    override fun init() {
        TODO("Not yet implemented")
    }

    override fun successResponse() {
        TODO("Not yet implemented")
    }

    override fun errorResponse() {
        TODO("Not yet implemented")
    }

}